src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -653,13 +658,27 @@ private static VersionInfo GetStopwrodsManagerVersionInfo()
                         ch.Warning("{0} should not be specified when default loader is TextLoader. Ignoring stopwordsColumn={0}",
                             stopwordsCol);
                     }
-                    loader = new SubComponent<IDataLoader, SignatureDataLoader>("TextLoader", "sep=tab col=Stopwords:TX:0");
+                    dataLoader = new TextLoader(
+                        env,
+                        new TextLoader.Arguments()
+                        {
+                            Separator = "tab",
+                            Column = new[]
+                            {
+                                new TextLoader.Column("Stopwords", DataKind.TX, 0)
```

Comment:
>"Stopwords" [](start = 54, length = 11)

Not in the scope of this change but I feel these names should be configurable from arguments instead of hardcoding where it might hide an existing column.

Reply:
I've opened [https://github.com/dotnet/machinelearning/issues/725](https://github.com/dotnet/machinelearning/issues/725) for this.

---
In reply to: [212152727](https://github.com/dotnet/machinelearning/pull/700#discussion_r212152727) [](ancestors = 212152727)